cluster/gce/util.sh

```diff
@@ -1303,7 +1303,17 @@ function test-teardown {
     --project "${PROJECT}" \
     --quiet \
     "${NODE_TAG}-${INSTANCE_PREFIX}-nodeports" || true
-  "${KUBE_ROOT}/cluster/kube-down.sh"
+  if [[ ${MULTIZONE:-} == "true" ]]; then
+      local zones=( $E2E_ZONES )
```

Comment:
ah, right. maybe `${E2E_ZONES[*]}`? bash arrays are always mysterious.

(similar nit in e2e-up.sh)